src/lavinmq/client/channel/stream_consumer.cr

```diff
@@ -34,7 +36,10 @@ module LavinMQ
             raise Error::PreconditionFailed.new("x-priority not supported on stream queues")
           end
           case frame.arguments["x-stream-offset"]?
-          when Nil, Int, Time, "first", "next", "last"
+          when Nil
```

Comment:
RabbitMQ says that if  `x-stream-offset` isn't specified it uses "next" as detault: 
> If it is unspecified the consumer will start reading from the next offset written to the log after the consumer starts

Reply:
Well, that quote is actually from the stream client, as rabbit doesn't support this for the amqp clients. 
So based on that we can do what we want. So if we have one way it works now, leave it at that for now might be good.